src/sqlfluff/dialects/dialect_ansi.py

```diff
@@ -3157,3 +3166,86 @@ class DatePartFunctionNameSegment(BaseSegment):
 
     type = "function_name"
     match_grammar = Sequence("DATEADD")
+
+
+@ansi_dialect.segment()
+class CreateTriggerStatementSegment(BaseSegment):
+    """Create Trigger Statement.
+
+    Taken from specification in https://www.postgresql.org/docs/14/sql-createtrigger.html
+    Edited as per notes in above - what doesn't match ANSI
+    """
+
+    type = "create_trigger"
+
+    match_grammar = Sequence("CREATE", "TRIGGER", Anything())
+
+    parse_grammar = Sequence(
+        "CREATE",
+        "TRIGGER",
+        Ref("TriggerReferenceSegment"),
+        OneOf("BEFORE", "AFTER", Sequence("INSTEAD", "OF")),
+        Delimited(
+            "INSERT",
+            "DELETE",
+            Sequence(
+                "UPDATE",
+                "OF",
+                Delimited(
+                    Ref("ColumnReferenceSegment"),
+                    terminator=OneOf("INSERT", "DELETE", "UPDATE", "ON"),
+                ),
+            ),
+            delimiter="OR",
```

Comment:
Not seen two character `delimiters` before and no test case for this. Could we add one just to make sure it works?

Reply:
Tested - it works!